app/src/main/res/values/strings.xml

```diff
@@ -20,4 +20,5 @@
     <string name="id_demo">1</string>
     <string name="songs_count">5 bài hát</string>
     <string name="album_name_demo">Narc Mp3</string>
+    <string name="name">name</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.name` appears to be unused
 Unused resources make applications larger and slow down builds.